locales/cy/translations.json

```diff
@@ -29,6 +29,12 @@
         "nameEnding" : "WELSH - Which private fund limited partnership name ending would you prefer to go on the public record?"
     },
 
+    "privateNameScotlandPage" : {
+        "title" : "WELSH - Scottish private fund limited partnership name",
```

Comment:
Note that I didn't add new texts for everything on the page - reused some from `namePage` and `privateNamePage` to avoid duplicates.

Reply:
all the other name templates seem to have their own translations. Name block will dissapear when all name templates are done though.

Reply:
Ok. Will need to decide if we share texts across the name pages or not. I won't merge this yet...

Reply:
Agreed to duplicate texts for now. Changes pushed.